### PR TITLE
Feature/66041

### PR DESCRIPTION
**Pull request by ofrankowska**
https://dev.snowdog.pro/issues/66041

### CODE REVIEW

File: Amazon_Payment/web/template/form/element/email.html [hunk 1]
```diff
@@ -51,32 +51,90 @@
...
+                        <svg
+                            class="
+                                icon
+                                icon--secondary
+                                input__button-pass-icon-view
+                            "
+                            role="presentation"
+                            focusable="false"
+                        >
+                            <title data-bind="i18n: 'View password icon'"></title>
```
Comment (pawel-snowdog) [inline Amazon_Payment/web/template/form/element/email.html:103]: can be removed if role="presentation" and focusable="false"

File: Amazon_Payment/web/template/form/element/email.html [hunk 2]
```diff
@@ -51,32 +51,90 @@
...
+                        <svg
+                            class="
+                                icon
+                                icon--secondary
+                                input__button-pass-icon-hide
+                            "
+                            role="presentation"
+                            focusable="false"
+                        >
+                            <title data-bind="i18n: 'Hide password icon'"></title>
```
Comment (pawel-snowdog) [inline Amazon_Payment/web/template/form/element/email.html:119]: can be removed

**Review by pawel-snowdog — APPROVED** [review]

File: Amazon_Payment/web/template/form/element/email.html (hunk 2 above)
Comment (anqaka) [reply Amazon_Payment/web/template/form/element/email.html]: 👍  done

File: Amazon_Payment/web/template/form/element/email.html (hunk 1 above)
Comment (anqaka) [reply Amazon_Payment/web/template/form/element/email.html]: 👍  done

**Pull request merged**